server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/repository/RepositoryController.java

```diff
@@ -0,0 +1,39 @@
+package de.tum.in.www1.hephaestus.codereview.repository;
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/repository")
```

Comment:
We can potentially just remove such code at the moment, if you don't want to maintain it. 

Right now we just need the schema and then have a controller for the leaderboard that we can fetch. No need to implement API endpoints that we might not use as of right now. But if you want to that's also fine